src/runtime-rs/crates/resource/src/share_fs/utils.rs

```diff
@@ -4,13 +4,27 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-use std::path::{Path, PathBuf};
+use std::{
+    os::unix::fs::PermissionsExt,
+    path::{Path, PathBuf},
+};
 
 use anyhow::Result;
 use kata_sys_util::mount;
 
 use super::*;
 
+pub(crate) fn mkdir_with_permissions(path_target: PathBuf, mode: u32) -> Result<()> {
+    let new_path = &path_target;
+    std::fs::create_dir_all(new_path)
+        .context(format!("unable to create new path: {:?}", new_path))?;
+
+    // mode 0o750 0o700
```

Comment:
I feel confused for the comment `mode 0o750 0o700`, can you give more detail;

Reply:
Thx.
The comment "mode format: 0o750, ..." maybe better. I will fix it.